orchestrator-salt/src/main/resources/salt/salt/unbound/init.sls

```diff
@@ -8,13 +8,19 @@ faulty_7_2_11_images_unbound_restart_patch:
     - repl: "pkill -u unbound -SIGHUP unbound"
     - ignore_if_missing: True
 
+{% if salt['pillar.get']('unbound:elimination:supported') == True %}
```

Comment:
please define a default value, so it won't fail in case the pillar is missing. False should be fine